framework/util/spirv_parsing_util.cpp

```diff
@@ -192,12 +192,29 @@ bool SpirVParsingUtil::ParseBufferReferences(const uint32_t* const spirv_code, s
 
     std::vector<Instruction> instructions;
 
-    // First build up instructions object to make it easier to work with the SPIR-V
+    bool found_buffer_ref = false;
+
+    // build up instructions object to make it easier to work with the SPIR-V
+    // also checks for required capability
     while (spirv_ptr < spirv_end)
     {
         Instruction& insn = instructions.emplace_back(spirv_ptr);
         spirv_ptr += insn.length();
         GFXRECON_ASSERT(insn.length() > 0);
+
+        if (insn.opcode() == spv::OpCapability && insn.word(1) == spv::CapabilityPhysicalStorageBufferAddresses)
+        {
+            found_buffer_ref = true;
+        }
+
+        if (insn.opcode() == spv::OpFunction)
```

Comment:
I would add a comment for the new-to-SPIRV that if we get to a `OpFunction` we know all the "metadata" of the spirv module

Reply:
done